ci_framework/roles/libvirt_manager/vars/main.yml

```diff
@@ -30,6 +30,7 @@ cifmw_libvirt_manager_dependency_packages:
   - qemu-kvm
   - libguestfs
   - guestfs-tools
+  - python-lxml
```

Comment:
isn't it `python3-lxml` ?

Reply:
`python-lxml` works fine because `python3-lxml` has a virtual provides, so this works: `dnf -y install python-lxml`.

But I'll change it for the sake of correctness.